AudioPlayer/AudioPlayer/AudioPlayer.swift

```diff
@@ -601,11 +601,69 @@ public class AudioPlayer: NSObject {
     - parameter time: The time to seek to.
     */
     public func seekToTime(time: NSTimeInterval) {
-        player?.seekToTime(CMTimeMake(Int64(time), 1))
+        let time = CMTime(seconds: time, preferredTimescale: 1000000000)
+        let seekableRange = player?.currentItem?.seekableTimeRanges.last?.CMTimeRangeValue
+        let seekableStart = seekableRange!.start
+        let seekableEnd = seekableRange!.end
+        
+        // check if time is in seekable range
+        if time >= seekableStart && time <= seekableEnd {
+            // time is in seekable range
+            player?.seekToTime(time)
+        } else if time < seekableStart {
```

Comment:
This isn't big issue but could you try to keep project's coding style for consistency?
```
if a {
}
else if b {
}
```